lib/Dialect/TritonGPU/IR/LinearLayoutConversions.cpp

```diff
@@ -239,6 +239,42 @@ LinearLayout sharedToLinearLayoutLeadingOffset(ArrayRef<int64_t> shape,
   return combineCtaCgaWithShape(tileLayout, shared.getCTALayout(), shape);
 }
 
+LinearLayout warpsDotOperand(MLIRContext *ctx, ArrayRef<unsigned> warpShape,
```

Comment:
@lezcano 
FYI, I've borrowed your `warpsNvidiaDot` function, because it perfectly fits MFMA/WMMA and in future blocked/FMA dot operand layouts